presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoUninstaller.java

```diff
@@ -70,9 +73,9 @@ private static void uninstallUsingRpm(String packageName, boolean checkDependenc
         }
```

Comment:
You should add return here. Because the success message should not be logged when Presto is running